utils/zpe-updater/util/canonicalstring_test.go

```diff
@@ -8,7 +8,6 @@ import (
 	"fmt"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
```

Comment:
why sure why this was removed and more interestingly how did the code compile??

Reply:
more interesting is that i didn't touched this file in my changes i will check it